bin/runtime/src/lib.rs

```diff
@@ -416,6 +433,51 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = pallet_multisig::weights::SubstrateWeight<Runtime>;
 }
 
+// We do not burn any money within treasury.
+pub const TREASURY_BURN: u32 = 0;
+// The percentage of the amount of the proposal that the proposer should deposit.
+// We agreed on non-progressive deposit.
+pub const TREASURY_PROPOSAL_BOND: u32 = 0;
+// The proposer should deposit max{`TREASURY_PROPOSAL_BOND`% of the proposal value, $10}.
+pub const TREASURY_MINIMUM_BOND: Balance = 1000 * CENTS;
+// Every 4h we implement accepted proposals.
+pub const TREASURY_SPEND_PERIOD: BlockNumber = 4 * HOURS;
+// We allow at most 20 approvals in the queue at once.
+pub const TREASURY_MAX_APPROVALS: u32 = 20;
+
+parameter_types! {
+    pub const Burn: Permill = Permill::from_percent(TREASURY_BURN);
+    pub const ProposalBond: Permill = Permill::from_percent(TREASURY_PROPOSAL_BOND);
+	pub const ProposalBondMinimum: Balance = TREASURY_MINIMUM_BOND;
```

Comment:
weird formatting?

Reply:
changed tabs to spaces (must have been introduced while copying from substrate :|)